tests/functional/lifecycle/LifecycleTask.js

```diff
@@ -344,6 +344,12 @@ class ProducerMock {
     }
 }
 
+class KafkaBacklogMetricsMock {
```

Comment:
looks like ProducerMock on L#305 just needs to add a mock of method `getKafkaProducer` to get these functional tests working again